src/server_manager/model/cloud.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright 2018 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export enum CloudProvider {
```

Comment:
Maybe call this CloudProviderId? It's unclear to me why you need that. A CloudProvider should have actions.

Reply:
I think what you call `CloudProvider`, I called `CloudProviderService` in the CLI. `CloudProviderId` and `CloudProvider` work just as well.